runtime/astar/Cargo.toml

```diff
@@ -9,78 +9,78 @@ build = "build.rs"
+cumulus-primitives-timestamp = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.16", default-features = false }
+cumulus-pallet-parachain-system = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.16", default-features = false }
+cumulus-pallet-aura-ext = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.16", default-features = false }
+parachain-info = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.16", default-features = false }
 
 # polkadot dependencies
-polkadot-parachain = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v0.9.13" }
-polkadot-runtime-common = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v0.9.13" }
+polkadot-parachain = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v0.9.16" }
+polkadot-runtime-common = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v0.9.16" }
 
 # benchmarking
 hex-literal = { version = '0.3.1', optional = true }
-frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false, optional = true }
-frame-system-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false, optional = true }
-pallet-collective = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false, optional = true }
+frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false, optional = true }
+frame-system-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false, optional = true }
+pallet-collective = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false, optional = true }
 
 # Astar pallets
-pallet-block-reward = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-block-reward-3.0.0/polkadot-0.9.13", default-features = false }
-pallet-collator-selection = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-collator-selection-3.0.0/polkadot-0.9.13", default-features = false }
-pallet-custom-signatures = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-custom-signatures-4.3.0/polkadot-0.9.13", default-features = false }
-pallet-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-dapps-staking-1.1.2/polkadot-0.9.13", default-features = false }
-pallet-vesting = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-vesting-4.0.0-dev/polkadot-0.9.13", default-features = false }
+pallet-block-reward = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.16", default-features = false }
```

Comment:
I really think we shouldn't do this and should go with tags instead.
This will become the new default branch and we have fairly often deliveries there.

Reply:
As I can see now, tags isn't so convenient as branches, you can't fork tag easily to provide new polkadot adjusted version.

Other point is to have default branch, if some should be fixed let's use dedicated branches. In situation with tags the new polkadot release is hell.

Reply:
We don't need to fork each tag separately - only what's needed.
The majority of pallets we use just take from the default - only `Astar` dapps-staking is special.
So when doing a polkadot upgrade, we could just:
* create a new branch for the new polkadot version and make this the new default branch
* create just a single new tag `polkadot-a.b.c` for this latest version
  * once we update a pallet, we can create the new tag as previously agreed 
* update runtimes to use this tag UNLESS they require a pallet version that differs from the latest default branch

It's not just about fixes though - the tags are useful for situations like we'll have when I merge `individual claim`.
Initially only Shibuya will use it - then Shiden - then Astar. I can work freely on the default branch in `astar-frame` as long as I know that Shiden and Astar don't depend on any branch and they use `dapps-staking-x.y.z./polkadot-a.b.c.` instead.